main.swift

```diff
@@ -65,6 +65,7 @@ class StringReplacer {
     private let authorEmail: String
     private let gitHubURL: String
     private let year: String
+    private let today: String
```

Comment:
Yes, `dateString` should be better. I just wanted to treat it the same as `year`